src/openrct2/world/Footpath.cpp

```diff
@@ -1064,6 +1064,7 @@ void footpath_chain_ride_queue(
 
         auto targetQueuePos = curQueuePos + CoordsDirectionDelta[direction];
         tileElement = map_get_first_element_at(targetQueuePos);
+        bool move = false;
```

Comment:
Please use a more descriptive identifier that explains what it means when `move` is true or false. From what I can see from quickly looking at the code, this is used to verify that a queue was found and can be connected.

Reply:
Yes, earlier in the code goto statement was used to connect it to queue but there is no such requirement we can just do this by using simple break statements and apply condition checkings at point.

Reply:
If that's the case, then a better identifier would be `foundQueue`, or something similar. Then the statement `if (!foundQueue)` makes immediate sense without having to know how the value was set.

Reply:
Noted. Will create a pr with foundQueue variable. Thank you

Reply:
This should be changed in this PR not in new one, this PR will not be merged unless we are happy with the code. You can just add more commits on top of this PR.